ani-cli

```diff
@@ -237,13 +250,14 @@ download() {
 }
 
 play_episode() {
+    [ "$skip_intro" = 1 ] && get_skip_times
     [ -z "$episode" ] && get_episode_url
     case "$player_function" in
         debug)
             [ -z "$ANI_CLI_NON_INTERACTIVE" ] && printf "All links:\n%s\nSelected link:\n" "$links"
             printf "%s\n" "$episode"
             ;;
-        mpv*) nohup "$player_function" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
+        mpv*) nohup "$player_function" "$skip_script" "$skip_argument" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
```

Comment:
Replace `"$skip_script" "$skip_argument"` with `$skip_flag`

Reply:
changed to:
```sh
mpv*) nohup "$player_function" "$skip_flag" "$skip_arg" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
```